storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -655,18 +555,22 @@ func addResourceTargetInfo(resource pcommon.Resource, settings Settings, earlies
 		// Do not pass identifying attributes as ignoreAttrs below.
 		identifyingAttrs = nil
 	}
-	labels := createAttributes(resource, attributes, scope{}, settings, identifyingAttrs, false, prompb.MetricMetadata{}, model.MetricNameLabel, name)
+	meta := metadata.Metadata{
+		Type: model.MetricTypeGauge,
+		Help: "Target metadata",
+	}
+	// TODO: should target info have the __type__ metadata label?
```

Comment:
On one hand it's simpler if it doesn't have __type__, since we'd have to filter it out when joining with other metrics.

On the other hand it's a bit inconsistent. I'd wait and see if we have to add it.

Reply:
Nice catch, would you like me to fix it in a separate PR and you rebase once merged, krajo?

Reply:
> Nice catch, would you like me to fix it in a separate PR and you rebase once merged, krajo?

Yes, I'll merge in from main.

Reply:
Alright, I did some investigation, and the current code does NOT add type/unit labels to target info.

We send empty metadata when [calling `createAttributes`](https://github.com/prometheus/prometheus/blob/25aee26a57ed08c8c4f518c56c834f6d2f567519/storage/remote/otlptranslator/prometheusremotewrite/helper.go#L682), and we [check that metadata exists](https://github.com/prometheus/prometheus/blob/25aee26a57ed08c8c4f518c56c834f6d2f567519/storage/remote/otlptranslator/prometheusremotewrite/helper.go#L207-L215) before adding those labels.


I debugged the behavior with Delve just to be sure:

<img width="962" height="182" alt="image" src="https://github.com/user-attachments/assets/00e7c5a0-e04c-456b-9346-8fe93bed04f7" />